install.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e
 
-MBGLVERSION='3.4.1'
+MBGLVERSION='3.5.0'
```

Comment:
> shouldn’t be necessary.

I should clarify “necessary”: the `release-v3.5.0` branch has the required changes to develop examples for that release. This PR should be (re)based on that branch, which eliminates the need to fiddle with `install.sh`.